src/backend/assets/dependencies/tweets.js

```diff
@@ -49,60 +49,64 @@ function lastTweet() {
-    }
-    interval = Math.floor(seconds / 3600);
-    if (interval > 1) {
-        return interval + " hours";
-    }
-    interval = Math.floor(seconds / 60);
-    if (interval > 1) {
-        return interval + " minutes";
-    }
-    return Math.floor(seconds) + " seconds";
+	if (interval > 1) {
+		return interval + " years";
+	}
+	interval = Math.floor(seconds / 2592000);
+	if (interval > 1) {
+		return interval + " months";
+	}
+	interval = Math.floor(seconds / 86400);
+	if (interval > 1) {
+		return interval + " days";
+	}
+	interval = Math.floor(seconds / 3600);
+	if (interval > 1) {
+		return interval + " hours";
+	}
+	interval = Math.floor(seconds / 60);
+	if (interval > 1) {
+		return interval + " minutes";
+	}
+	return Math.floor(seconds) + " seconds";
 }
 
 function parser(data) {
-
-	var parsed = "";
-	var tweet = data.statuses[tweetNum].text;
-	var words = tweet.split(" ");
-	var loklakLinkCount = 0;
-	for (var word in words) {
-		if (words[word].startsWith("@")) {
-			parsed += "<a href='https://twitter.com/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
-		} else if (words[word].startsWith("#")) {
-			parsed += "<a href='https://twitter.com/hashtag/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
-		} else if (words[word].startsWith("http")) {
-			if (words[word].startsWith("http://loklak")) {
-				parsed += "<a href='" + data.statuses[tweetNum].links[loklakLinkCount] + "' target='_blank'>" + data.statuses[tweetNum].links[loklakLinkCount] + "</a> ";
-				loklakLinkCount += 1;
+	try{
+		var parsed = "";
+		var tweet = data.statuses[tweetNum].text;
+		var words = tweet.split(" ");
+		var loklakLinkCount = 0;
+		for (var word in words) {
+			if (words[word].startsWith("@")) {
+				parsed += "<a href='https://twitter.com/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
+			} else if (words[word].startsWith("#")) {
+				parsed += "<a href='https://twitter.com/hashtag/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
+			} else if (words[word].startsWith("http")) {
+				if (words[word].startsWith("http://loklak")) {
+					parsed += "<a href='" + data.statuses[tweetNum].links[loklakLinkCount] + "' target='_blank'>" + data.statuses[tweetNum].links[loklakLinkCount] + "</a> ";
+					loklakLinkCount += 1;
+				} else {
+					parsed += "<a href='" + words[word] + "' target='_blank' style='word-break:break-all'>" + words[word] + "</a> ";
+				}
 			} else {
-				parsed += "<a href='" + words[word] + "' target='_blank' style='word-break:break-all'>" + words[word] + "</a> ";
+				parsed += words[word] + " ";
 			}
-		} else {
-			parsed += words[word] + " ";
 		}
+		document.getElementById("tweet").innerHTML = parsed;
+		var user = data.statuses[tweetNum].user;
+		var tweetDate = new Date(data.statuses[tweetNum].created_at);
+		document.getElementById("dateTweeted").innerHTML = "Tweeted " + timeSince(tweetDate) + " ago";
```

Comment:
@aayusharora Sorry, I didn't understand well 😞 , Where do I have to save the variables?